libs/messaging/kafka-message-bus-impl/build.gradle

```
@@ -16,6 +16,7 @@ dependencies {
     implementation project(":libs:configuration:configuration-merger")
     implementation project(":libs:metrics")
     implementation project(":libs:utilities")
+    implementation project(":libs:serialization:serialization-avro")
```

```suggestion
    api project(":libs:serialization:serialization-avro")
```
Due to `CordaAvroSerializationFactoryImpl` using `CordaAvroSerializationFactory` as a super-type.